src/DockerizeManager.js

```diff
@@ -0,0 +1,109 @@
+const NODE_RELEASE = 'carbon'; // Node release used in Dockerfiles
+
+var utils = require('./utils'),
+    rm_rf = require('rimraf'),
+    plural = require('plural'),
+    fs = require('fs'),
+    Q = require('q'),
+    path = require('path'),
+    ejs = require('ejs'),
+    exists = require('exists-file'),
+    Logger = require(__dirname + '/Logger');
+
+
+const OUT_FILES = {
+    DEFAULT: [
+        './config/config.docker.js',
+        './Dockerfile',
+        './DockerfilePluginWorker',
+        './docker-compose.yml'
+    ],
+    PRODUCTION: [
+        './config/config.dockerprod.js',
+        './DockerfileNginx',
+        './docker-compose-prod.yml',
+        './nginx.conf'
+    ],
+};
+
+function DockerizeManager(logger) {
+    this._logger = logger || new Logger();
+}
+
+
+DockerizeManager.prototype._go_to_root_dir = function() {
```

Comment:
If you inherit from the ComponentManager (like [here](https://github.com/webgme/webgme-cli/blob/master/src/ComponentManager.js#L33)) then you can just use the `_preprocess` fn to wrap it and won't need to rewrite the preprocessing fn-ality. This basically ensures that functions that need to run within an existing webgme app do.

Reply:
I didn't think this command fit within Components so I decided to keep the implementation separate.

Reply:
That makes sense but I am still torn bc I don't like having two implementations for the same thing...

What if you just "inherited" that method? Something like:
```
DockerizeManager.prototype._preprocess = ComponentManager.prototype._preprocess;
```

In the future, we could more `_preprocess` to be a utility rather than a method defined by the Component base class.

Reply:
I did look into that option. However _preprocess does more than just navigate to the root-dir. It process `webgme-setup.json`. 
I could, however, extract a util function `changeDirToRoot` (`_go_to_root_dir` no idea why I made this into `snake_case` btw).